invenio/b2share/modules/b2deposit/b2share_model/model.py

```diff
@@ -140,10 +139,8 @@ def __init__(self):
                            'were relevant in the creation of the resource.'
         }
         self.field_args['language'] = {
-            'hidden': True,
             'value': self.language_default,
-            # 'description':
-            # 'The name of the language the document is written in.'
+            'description': 'The name of the language the document is written in.'
```

Comment:
The Linguistics domain also has a language_code field. We must avoid having 2 different fields for the same thing, especially if they can be filled in by the user with contradictory information.